services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/cli.py

```diff
@@ -52,6 +52,22 @@ def _print_highlight(message: str) -> None:
     typer.echo(typer.style(message, fg=typer.colors.MAGENTA))
 
 
+@main.command()
+def state_list_dirs():
+    """Lists files inside state directories"""
+
+    async def _async_state_list_dirs() -> None:
+        app = await _setup_app_for_task_execution()
+
+        mounted_volumes: MountedVolumes = app.state.mounted_volumes
+
+        for state_path in mounted_volumes.state_paths:
+            state_path_content = list(state_path.glob("*"))
+            typer.echo(f"Entries in {state_path}: {state_path_content}")
+
+    asyncio.get_event_loop().run_until_complete(_async_state_list_dirs())
```

Comment:
shoudn't we use ``asyncio.run`` now? SEE https://docs.python.org/3/library/asyncio-task.html#asyncio.run

Reply:
👍 Thanks, was searching for this the last time but I could not find it!